autoscheduler/scraper/models/section.py

```diff
@@ -1,9 +1,43 @@
-from django.db import models
+"""Models for meetings and sections.
+Together, they represent all the information for a particular meeting time.
+"""
 
-class Meeting(models.Model):
-    """ Describe model here """
+from django.contrib.postgres.fields import ArrayField
+from django.db import models
+from scraper.models import Instructor
 
 class Section(models.Model):
-    """ Section will contain Meeting objects,
-        which is why its placed below it
+    """ Section contains data for a group of meetings.
+    """
+    id = models.IntegerField(primary_key=True) # id is primary key in scraped data
+    subject = models.CharField(max_length=4, db_index=True)
+    course_num = models.IntegerField(db_index=True)
+    section_num = models.IntegerField(db_index=True)
+    term_code = models.IntegerField(db_index=True)
+    min_credits = models.IntegerField()
+    max_credits = models.IntegerField(null=True) #min_credits is never null, but max_credits can be
+    max_enrollment = models.IntegerField()
+    current_enrollment = models.IntegerField()
+    instructor = models.ForeignKey(Instructor, on_delete=models.CASCADE)
+
+    class Meta:
+        db_table = "sections"
+
+class Meeting(models.Model):
+    """Describes a particular meeting time of a section.
+    Each section has one or more meetings.
     """
+    id = models.IntegerField(primary_key=True) # id is primary key in scraped data

```

Comment:
```suggestion
    id = models.BigIntegerField(primary_key=True) # id is primary key in scraped data
```

Needs to be a `BigIntegerField`, as  `41266-201931-0` converted to an Integer & removing the redundant `20` in `2019` is 4.1 billion

Reply:
We could also make it a CharField so we can just leave it with the hypens instead of having to manipulate the strings, it would definitely be faster when we're scraping and probably the same performance doing queries, but I'm not sure honestly